.github/workflows/distcheck.yaml

```diff
@@ -212,7 +219,8 @@ jobs:
         run: |
           set -x
 
-          export CC="${{ matrix.realcc || matrix.cc }}"
+          export CC="cc"
+          export CXX="c++"
```

Comment:
I'd have a slight preference to setting these in the configure line, so that it's obvious from the output what CC and CXX were set to.  `./configure CC=cc CXX=c++ --with-mpi=....` will work well, and saves the values in the right place.

Reply:
I have seen in the past that configuring with `./configure CC=/path/to/specific/compiler` or `./configure CFLAGS="xyz"` does not properly cache the value in all cases, and I've seen that if I later run `make` without passing the same `CC=/path/to/specific/compiler` or `CFLAGS="xyz"`, I can get build failures and inconsistent flags.

I believe it's something around our modification of CC for MPICC [here](https://github.com/aws/aws-ofi-nccl/blob/master/tests/functional/Makefile.am#L11). Rather than investigating, I've just started exporting them.

Reply:
That's a bug, not something to work around.